vllm/_core_ext.py

```diff
@@ -131,10 +131,37 @@ def is_ieee_754(self) -> bool:
                 not self._finite_values_only
 
         def __str__(self) -> str:
-            raise NotImplementedError
+            """
+            naming generally follows: https://github.com/jax-ml/ml_dtypes
```

Comment:
I updated this class from the original repo because it blocked me from testing the feature. The class is defined here:  https://github.com/vllm-project/vllm/blob/fb2716d64117aaa6c36b97b09765aa10a89e2fe5/vllm/scalar_type.py#L19

Let me know if there is a better way,

Reply:
You are right. Vllm was rebased and this methods do have those defitniitions. Please rebase